### DISCUSSION TITLE
Does the Antigen-antibody benchmark use MSA ?

### DISCUSSION THREAD

**Discussion by akshay-chenna** [chaidiscovery/chai-lab#222] [General]
Hi,

I am not clear from this X [post](https://x.com/chaidiscovery/status/1859025154682347569) if Chai-1r was run with or w/o MSAs for the antibody-antigen prediction with restraints. Could someone please clarify?

Thanks,

Akshay

**Comment by jackdent** [comment]
We ran the benchmark with MSAs

**Reply by akshay-chenna** [reply]
Thank you!
